popde/adaptive_kde.py

```diff
@@ -224,6 +224,147 @@ def optimize_parameters(self, method='loo_cv', fom_plot_name=None):
         return best_params, best_score
 
 
+class KDERescaleOptimization(AdaptiveBwKDE):
+    """
+     A class to optimize rescaling factors for each dimension and 
+      [an optional alpha parameter]
+    using Nelder-Mead optimization. 
+    The optimization is conducted by minimizing a cross-validation
+    score, based on log-likelihood as the figure of merit.
+
+    Attributes:
+        alpha (float): Initial alpha parameter for rescaling.
+        bandwidth (float): Fixed bandwidth, default set to 1.0.
+        n_splits (int): Number of splits for k-fold cross-validation.
+
+    Methods:
+        loo_cv_score(rescale_factors_alpha): 
```

Comment:
it's not necessary to include the arguments to the method, just leave them out
```suggestion
        loo_cv_score
```

Reply:
fixed